marshmallow/fields.py

```diff
@@ -607,6 +607,9 @@ class Boolean(Field):
     def _serialize(self, value, attr, obj):
         if value is None:
             return None
+        elif value in self.falsy:
```

Comment:
Also, can you update the comments on ~L601 and ~L604 to reflect this change? Should just be a matter of changing "deserialize" to "(de)serialize".